crates/re_sdk/src/recording_stream.rs

```diff
@@ -1543,7 +1543,7 @@ impl RecordingStream {
         &self,
         addr: std::net::SocketAddr,
         flush_timeout: Option<std::time::Duration>,
-        blueprint: Option<Vec<LogMsg>>,
+        blueprint: Option<(Vec<LogMsg>, BlueprintReadyOpts)>,
```

Comment:
At this point maybe we should have

``` rs
struct SerializedBlueprint {
    messages: Vec<LogMsg>,
    on_load: BlueprintReadyOpts,
}
```

Reply:
Ahh right meant to do something like that. Though I might get rid of this mechanism all together and **just** use the new `send_blueprint` API, which doesn't have the same issue since both fields are required in that case.